src/components/UI/AttachmentLink/AttachmentLink.tsx

```diff
@@ -37,7 +52,14 @@ export const AttachmentLinkUnStyled = ({ className, filename, data, type, path }
             <span className="filename">{filename}</span>
             {hasBase64 && <span className="filesize">({filesize})</span>}
           </p>
-          <p className="downloadtext mb-0">Download</p>
+          <div className="row no-gutters">
+            <p className="downloadtext mb-0">Download</p>
+            {redirectLink && (
+              <a href={redirectLink} target="_blank" rel="noopener noreferrer" className="downloadtext mb-0 ml-2">
+                Open
+              </a>
+            )}
```

Comment:
this a tag is currently nested within another a tag (not valid) u may wanna move the parent download href to the download text instead? sorry about that didn't for-see this

Reply:
Yeah i just notice the warning too hahah 👍